docs/source/onnx.rst

```diff
@@ -872,6 +872,101 @@ The interface for specifying operator definitions is experimental;
 adventurous users should note that the APIs will probably
 change in a future interface.
 
+Autograd Function
```

Comment:
If possible could you rebase, I'm not exactly sure which changes are related to this PR

Reply:
Thanks, done